src/pages/get-started/commands.md

```diff
@@ -88,6 +88,44 @@ Configured event provider details:
 - description: testing local provider creation (Instance docs-stage-testing)
 ```
 
+## Get details about configured event registrations in your App Builder application
```

Comment:
Add a link to this section at about line 23 of this file.

Reply:
Added